qa/cephfs/clusters/2-mds-2-client.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- [mon.a, mgr.y, mds.a, mds.c, osd.0, osd.1, osd.2, osd.3]
+- [mon.b, mon.c, mgr.x, mds.b, mds.d, osd.4, osd.5, osd.6, osd.7]
+- [client.0]
+- [client.1]
+openstack:
+- volumes: # attached to each instance
+    count: 4
+    size: 20 # GB
+- machine:
+    disk: 200 # GB
+log-rotate:
+  ceph-mds: 10G
+  ceph-osd: 10G
```

Comment:
Is this a leftover? We only need one cluster definition?

Reply:
It wasn't a left over. Anyways, since qa reorg no longer need this.